### PR TITLE
Add csslint check on travis

### PR DESCRIPTION
**Pull request by nicolas-joubert**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,8 +7,6 @@ php:
   - 7.0
   - hhvm
 
-sudo: false
```
Comment (soullivaneuh) [inline .travis.yml:10]: We can't enable sudo. It will break cache system.

File: .travis.yml [hunk 2]
```diff
@@ -24,6 +22,8 @@ matrix:
   include:
     - php: 5.6
       env: CS_FIXER=run
+    - php: 5.6
+      env: CSSLINT=run
```
Comment (soullivaneuh) [inline .travis.yml:26]: Maybe we should having only one variable like `CODE_CHECK` for example.

**Comment by soullivaneuh** [comment]
I think you made a mistake. Your PR is on your fork, not on Sonata! ;-)

**Comment by nicolas-joubert** [comment]
It's a temporary PR on my previous work, that's why it's on my fork.

But, "We can't enable sudo. It will break cache system." 
So it wasnt' possible to install node & csslint => PR closed

**Pull request closed without merging**